src/schema/filter_artworks.ts

```diff
@@ -103,38 +103,58 @@ export const FilterArtworksType = new GraphQLObjectType<any, ResolverContext>({
       resolve: (
         { options: gravityOptions },
         args,
-        { filterArtworksLoader }
+        {
+          unauthenticatedLoaders: { filterArtworksLoader: loaderWithCache },
+          authenticatedLoaders: { filterArtworksLoader: loaderWithoutCache },
+        }
       ) => {
         const relayOptions = convertConnectionArgsToGravityArgs(args)
+        const {
+          include_artworks_by_followed_artists,
+          aggregations,
+        } = gravityOptions
+        const requestedPersonalizedAggregation = aggregations.includes(
+          "followed_artists"
+        )
 
-        return filterArtworksLoader(
-          assign(gravityOptions, relayOptions, {})
-        ).then(({ aggregations, hits }) => {
-          if (!aggregations || !aggregations.total) {
-            throw new Error("This query must contain the total aggregation")
-          }
+        let loader
+        if (
+          include_artworks_by_followed_artists ||
+          requestedPersonalizedAggregation
+        ) {
+          loader = loaderWithoutCache
+        } else {
+          loader = loaderWithCache
+        }
 
-          const totalPages = computeTotalPages(
-            aggregations.total.value,
-            relayOptions.size
-          )
+        return loader(assign(gravityOptions, relayOptions, {})).then(
+          ({ aggregations, hits }) => {
+            if (!aggregations || !aggregations.total) {
+              throw new Error("This query must contain the total aggregation")
+            }
 
-          return assign(
-            {
-              pageCursors: createPageCursors(
-                relayOptions,
-                aggregations.total.value
-              ),
-            },
-            connectionFromArraySlice(hits, args, {
-              arrayLength: Math.min(
-                aggregations.total.value,
-                totalPages * relayOptions.size
-              ),
-              sliceStart: relayOptions.offset,
-            })
-          )
-        })
+            const totalPages = computeTotalPages(
+              aggregations.total.value,
+              relayOptions.size
+            )
+
+            return assign(
```

Comment:
Nit-pick You can just replace this with object spread or `Object.assign()` no need any longer for lodash.